yt_dlp/extractor/youtube.py

```diff
@@ -3661,6 +3676,9 @@ def _extract_player_response(self, client, video_id, master_ytcfg, player_ytcfg,
             'videoId': video_id,
         }
 
+        if _split_innertube_client(client)[0] == 'android_testsuite':
+            yt_query['params'] = '2AMB'
```

Comment:
perhaps define client-specific params in the client definitions?

think I was originally planning on doing that at one stage